src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-eks-add-on.mdx

```diff
@@ -0,0 +1,108 @@
+## This document shows the steps required to use EKS Add-On from AWS Marketplace.
+#### **NOTE : This add-on is currently only supported on x86 instance types, with support for ARM64 coming soon**
+
+## Prerequisites
+1. [Amazon Elastic Kubernetes Service (EKS)](https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html) Cluster. See the userguide here
+2. A valid New Relic account. You can subscribe to New Relic from AWS Marketplace with Free Tier. See details [here](https://aws.amazon.com/marketplace/pp/prodview-ov56chowabeb4?sr=0-3&ref_=beagle&applicationId=AWS-Marketplace-Console)
+3. Subscribe to [New Relic Kubernetes Integration](https://aws.amazon.com/marketplace/pp/prodview-gcywa6keq2ajy?applicationId=AWS-Marketplace-Console&ref_=beagle&sr=0-5), available for free
+4. Login to [New Relic](https://one.newrelic.com/) and generate a [License Key](https://one.newrelic.com/api-keys)
+
+## Provision New Relic Kubernetes Operator EKS Add-On via AWS console
+1. Navigate to the EKS Cluster > Add-Ons > Get More add-ons
+    <img
+      title="eksAddOnTab.png"
+      alt="EKS Add On Tab."
+      src={eksAddOnTab}
+    />
+2. Under AWS Marketplace add-on, search, browse or navigate to New Relic Kubernetes Integration, select the checkbox and click Next
+    <img
+      title="eksAddOnMarketplace.png"
+      alt="EKS Add On Marketplace."
+      src={eksAddOnMarketplace}
+    />
+3. Select the latest default version or choose an alternate version, if required, and click Next
+    <img
+      title="eksAddOnMarketplaceConfig.png"
+      alt="EKS Add On Marketplace Config"
+      src={eksAddOnMarketplaceConfig}
+    />
+4. Click Create
+
+5. Within a few minutes, New Relic's Kubernetes Operator should be up and running. Please view the configuration steps below to collect and ship observability data from your EKS Cluster to New Relic.
+
+
+# Provision NewRelic EKS Add-On via AWS Command Line Interface (CLI)
+
+1. Execute the following command to install the NewRelic Add-On
+   ```
+   aws eks create-addon --cluster-name REPLACE_ME --addon-name new-relic_kubernetes-operator
+   ```
+where --cluster-name : Name of your EKS Cluster
+
+Expected Output
+
+Output will be similar to following
+    ```
+    {
+        "addon": {
+            "addonName": "new-relic_kubernetes-operator",
+            "clusterName": "newrelic-test",
+            "status": "CREATING",
+            "addonVersion": "v0.1.8-eksbuild.1",
+            "health": {
+                "issues": []
+            },
+            "addonArn": "arn:aws:eks:ap-northeast-2:292226546026:addon/kubecon2023-127/new-relic_kubernetes-operator/6cc5e7b7-d8f1-5fa0-c668-2bc9727e16b3",
+            "createdAt": "2023-07-27T11:47:57.023000-07:00",
+            "modifiedAt": "2023-07-27T11:47:57.044000-07:00",
+            "tags": {}
+        }
+    }
+    ```
+
+** New Relic Kubernetes Operator Configuration **
+
+You may confirm the Kubernetes Operator Running by executing kubectl get all -n newrelic.
```

Comment:
Should be You may confirm that the Kubernetes Operator is running by executing "kubectl get all -n newrelic" at the CLI command line.